src/core/core.tooltip.js

```diff
@@ -62,11 +66,13 @@ defaults._set('global', {
 
 			// Args are: (tooltipItem, data)
 			beforeLabel: helpers.noop,
-			label: function(tooltipItem, data) {
+			label: function(tooltipItem, data, mode) {
 				var label = data.datasets[tooltipItem.datasetIndex].label || '';
-
 				if (label) {
 					label += ': ';
+					if (mode === 'dataset') {
+						label = tooltipItem.xLabel + ': ';
```

Comment:
It'd be better to move this to where the `label` variable is declared like:

    var label = (mode === 'dataset' ? tooltipItem.xLabel : data.datasets[tooltipItem.datasetIndex].label) || '';